bsmetadata/train.py

```diff
@@ -156,10 +195,26 @@ def main(args: CFG) -> None:
     else:
         args.num_train_epochs = math.ceil(args.max_train_steps / num_update_steps_per_epoch)
 
-    if args.num_eval < 1:
+    if args.evaluation_strategy == IntervalStrategy.EPOCH:
+        if args.eval_num_per_epoch < 1:
+            eval_per_n_step = args.max_train_steps + 1
```

Comment:
@tianjianjiang, are you referring to this kind of `+1` ? :slightly_smiling_face:

Reply:
@SaulLu Yes~

Reply:
I just kept what was done before, and it seems to me that this has the effect of not doing an evaluation. :slightly_smiling_face:

Reply:
Can confirm, I did it in the first place, I should've added a comment `# will never eval`. 🙃